internal/compiler/widgets/cupertino-base/scrollview.slint

```diff
@@ -113,6 +115,16 @@ export component ScrollView {
         maximum:  i-flickable.viewport-height - i-flickable.height;
         page-size:  i-flickable.height;
         visible: i-flickable.viewport-height > i-flickable.height;
+
+        scroll-event(event) => {
+            if (event.delta-y != 0) {
+                i-flickable.viewport-y =
```

Comment:
I still feel that `ScrollBar` should have built-in `scroll-event` handling. It has its own buttons, its own mouse handling, but so IMO it should also have the scroll event handling. This might require adding an `orientation` property, but we have an enum for that now :)

Reply:
changed.